docs/install.md

```diff
@@ -33,12 +33,32 @@ For credential-based logins such as Azure Container Registry, Docker Hub and Qua
 $ echo '{"username":"jdoe","password":"secret","email":"jdoe@acme.com"}' | base64
 ```
 
-For token-based logins such as Google Container Registry and Amazon ECR, use:
+For token-based logins including Amazon ECR, use:
 
 ```
 $ echo '{"registrytoken":"9cbaf023786cd7"}' | base64
 ```
 
+### For Google Container Registry:
```

Comment:
Would you mind making the other topics like token-based auth a header as well for consistency?